ramls/eholdings.raml

```diff
@@ -94,3 +94,14 @@ types:
     responses:
       204:
         description: No Content
+
+/loadHoldings:
+  displayName: Run load holdings job
+  post:
+    description: Run load holdings job.
+    headers:
+      Content-Type:
+        example: application/json
+    responses:
+      204:
+        description: No Content
```

Comment:
Thinking of debugging/troubleshooting this task (since it has multiple steps with potentials for issues/errors along the way - wondering if a separate endpoint for diagnostic information may be helpful. Realize there is a holdingsIq endpoint to check status and our log files/database (but not always easily accessible)

Reply:
> 
> 
> Thinking of debugging/troubleshooting this task (since it has multiple steps with potentials for issues/errors along the way - wondering if a separate endpoint for diagnostic information may be helpful. Realize there is a holdingsIq endpoint to check status and our log files/database (but not always easily accessible)

Agree. It's not easy to understand the progress and final result of data loading. Definitely a point to improve in future